File: IronVault/UI/ironvault_desktop/lib/models/trener.dart
import 'package:json_annotation/json_annotation.dart';

part 'trener.g.dart';

@JsonSerializable()
class Trener {
  int? trenerId;
  String? ime;
  String? prezime;
  String? email;
  String? brojTelefona;
  String? slika;

  Trener({this.trenerId});

  factory Trener.fromJson(Map<String, dynamic> json) =>
      _$TrenerFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$TrenerToJson(this);
}

File: IronVault/IronVault.Model/Models/Trener.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IronVault.Model.Models
{
    public partial class Trener
    {
        public int TrenerId { get; set; }

        public string Ime { get; set; } = null!;

        public string Prezime { get; set; } = null!;

        public string Email { get; set; } = null!;

        public string BrojTelefona { get; set; } = null!;

        public byte[]? Slika { get; set; }
    }
}
